pkg/controller/servicebindingrequest/binder.go

```diff
@@ -67,23 +65,26 @@ var EmptyApplicationSelectorErr = errors.New("application ResourceRef or MatchLa
 var ApplicationNotFound = errors.New("Application is already deleted")
 
 // search objects based in Kind/APIVersion, which contain the labels defined in ApplicationSelector.
-func (b *Binder) search() (*unstructured.UnstructuredList, error) {
+func (b *Binder) search() ([]*unstructured.Unstructured, error) {
```

Comment:
Is this change from `*unstructured.UnstructuredList` to `[]*unstructured.Unstructured` essential for this change?